components/eam/src/physics/cam/gw_drag.F90

```diff
@@ -88,6 +88,9 @@ module gw_drag
   ! Convective heating rate conversion factor, default is 20.0_r8
   real(r8) :: gw_convect_hcf = 20.0_r8
 
+  ! Scaling factor for the heating depth
+  real(r8) :: hdepth_scaling_factor = 1
```

Comment:
Since it is `real(r8)`, it would be better to initialize it with `_r8` (`real(r8):: hdepth_scaling_factor = 1.0_r8`). Maybe it doesn't matter but it will be good for consistency.

Reply:
I agree, best to add `_r8`

Reply:
I thought it was better to omit it since the compiler will ensure that the type matches?